src/common/config.rs

```diff
@@ -3,20 +3,36 @@ use std::{fs, path::Path};
 use toml;
 
 use crate::common::{token::check_valid_tokens, KoraError};
+use crate::common::instructions::{get_program_instruction_configs_with_discriminators, ProgramInstructionConfig, ProgramInstructionConfigWithDiscriminators};
 use solana_client::nonblocking::rpc_client::RpcClient;
 
 #[derive(Debug, Deserialize)]
 pub struct Config {
-    pub validation: ValidationConfig,
+    pub validation: RawValidationConfig,
```

Comment:
if there's a better rust way to do this lmk. Basically want one version of this struct before validation and another validation as we add additional metadata based on the IDL parsed in the validation step